presto-main/src/main/java/com/facebook/presto/server/security/LdapServerConfig.java

```diff
@@ -20,7 +20,16 @@
     private String ldapUrl;
     private String baseDistinguishedName;
     private boolean authenticationEnabled;
+    private String serverType = ServerType.AD.toString();
```

Comment:
yup this will be optional and AD will be default value. Only if a different serverType is needed user will set this.